### ISSUE TITLE
oc create -f error: yaml: line 601: did not find expected key

### ISSUE THREAD

**Issue by jludvice** [syndesisio/openshift-templates#19]
oc create -f https://raw.githubusercontent.com/redhat-ipaas/openshift-templates/master/redhat-ipaas.yml 

error: yaml: line 601: did not find expected key

**Comment by chirino** [comment]
upgrade your oc client to at least 1.4.1

**Comment by jludvice** [comment]
```
oc version
oc v1.5.0-rc.0+49a4a7a
kubernetes v1.5.2+43a9be4
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://192.168.1.13:8443
openshift v1.5.0-rc.0+49a4a7a
kubernetes v1.5.2+43a9be4
```

**Comment by iocanel** [comment]
Same version, same issue

**Comment by chirino** [comment]
Should be fixed in 4eff47b6ab5b1d0c2ddb9f8082cce08f130e7a41
